### ISSUE TITLE
What about async fuckit?

### ISSUE THREAD

**Issue by nvaytet** [ajalt/fuckitpy#47]
Does it exist? Is it needed?
`fuckit` is awesome :-)

**Comment by pomponchik** [comment]
No, but there is a similar library with fully support of async - [`exception_escaping`](https://github.com/pomponchik/exception_escaping).
